Resizin/Classes/ResizinSettings.swift

```diff
@@ -165,6 +180,8 @@ public struct ResizinSettings {
         if let upscale = upscale { modifiers.append("u_\(upscale ? 1 : 0)")}
         if let background = background { modifiers.append("bg_\(background)\(alpha)") }
         if let border = border { modifiers.append("b_\(border.top)_\(border.left)_\(border.bottom)_\(border.right)") }
+        if let outputFormat = outputFormat {
+            modifiers.append("o_\(outputFormat)") }
```

Comment:
it'd be nice to have tests for this, but as there are currently no tests in this repo, it's 🤷 I guess

Reply:
Maybe next time. 🙈